### PR TITLE
feat: add playground

### PR DESCRIPTION
**Pull request by labbomb**
### Purpose
New Playground module
image

### CODE REVIEW

File: paimon-web-ui-new/src/components/monaco-editor/index.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        target: monaco.languages.typescript.ScriptTarget.ES2020,
+        allowNonTsExtensions: true
+      })
+      editor = monaco.editor.create(codeEditBox.value, {
+        value: props.modelValue,
+        language: props.language,
+        theme: monacoEditorThemeRef.value,
+        ...props.options
+      })
+      // 监听值的变化
```
Comment (songjianet) [inline paimon-web-ui-new/src/components/monaco-editor/index.tsx:70]: use english

File: paimon-web-ui-new/src/components/monaco-editor/index.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    )
+    watch(
+      () => props.language,
+      newValue => {
+        monaco.editor.setModelLanguage(editor.getModel()!, newValue)
+      }
+    )
+
+    watch(
+      () => configStore.getCurrentTheme ,
```
Comment (songjianet) [inline paimon-web-ui-new/src/components/monaco-editor/index.tsx:104]: Is there a space before the comma?

File: paimon-web-ui-new/src/views/playground/index.tsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+              tab: () => (
+                <n-icon size="24">
+                  <Layers />
+                </n-icon>
+              )
+            }}
+          >
+            <div class={styles.content}>
+              <div class={styles.catalog}>
+                <CataLog></CataLog>
```
Comment (songjianet) [inline paimon-web-ui-new/src/views/playground/index.tsx:64]: <CataLog />

File: paimon-web-ui-new/src/views/playground/index.tsx [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+              </div>
+              <div class={styles.editor}>
+                <MonacoEditor
+                  v-model={this.content}
+                  language={this.language}
+                  onEditorMounted={this.editorMounted}
+                  ></MonacoEditor>
```
Comment (songjianet) [inline paimon-web-ui-new/src/views/playground/index.tsx:71]: <MonacoEditor />

**Review by songjianet — CHANGES_REQUESTED** [review]

File: paimon-web-ui-new/src/views/playground/index.tsx (hunk 3 above)
Comment (songjianet) [reply paimon-web-ui-new/src/views/playground/index.tsx:64]: <CataLog />

File: paimon-web-ui-new/src/locales/zh/modules/playground.ts [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License. */
+
+export default {
+  select_catalog: '选择Catalog',
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/locales/zh/modules/playground.ts:19]:
```suggestion
  select_catalog: '选择 Catalog',
```

File: paimon-web-ui-new/src/components/monaco-editor/type.ts [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+        },
+        readOnly: false,
+        contextmenu: true,
+        fontSize: 16,
+        scrollBeyondLastLine: false,
+        overviewRulerBorder: false
+      }
+    }
+  }
+}
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/components/monaco-editor/type.ts:79]: New line for the end of file.

File: paimon-web-ui-new/src/router/routes.ts [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+/**
+ * Basic page
+ */
+const basePage: RouteRecordRaw[] = [
+  ...playground_routes,
+]
+
+/**
+   * Login page
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/router/routes.ts:29]: The indent of comment is incorrect.

File: paimon-web-ui-new/src/views/playground/index.tsx [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+
+export default defineComponent({
+  name: 'PlaygroundPage',
+  setup() {
+    const type = ref<TabsProps['type']>('bar')
+
+    const content = ref('')
+    const language = ref('javascript')
+    const editorMounted = (editor: monaco.editor.IStandaloneCodeEditor) => {
+      console.log('editor实例加载完成', editor)
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/views/playground/index.tsx:34]:
```suggestion
      console.log('Loaded editor instance.', editor)
```

**Review by SteNicholas — CHANGES_REQUESTED** [review]
@labbomb, thanks for the contribution. I have left some comments for this pull request. PTAL.

**Review by SteNicholas — APPROVED** [review]
LGTM. Thanks for @labbomb updates.

**Pull request merged**